components/Navigation/index.js

```diff
@@ -0,0 +1,38 @@
+import styled from "styled-components";
+import Link from "next/link.js";
+
+const List = styled.ul`
+  list-style: none;
+  display: flex;
+  background-color: #584849;
+  border: 3px solid black;
+  bottom: 0px;
+  right: 0px;
+  left: 0px;
+  justify-content: space-around;
+  padding: 1rem;
+`;
+
+const NavLink = styled(Link)`
+  text-decoration: none;
+  color: black;
+  font-weight: bold;
+  &:hover {
+    background-color: #a48476;
+  }
+`;
+
+export default function Navigation() {
+  return (
+    <nav>
```

Comment:
It could be a good idea to style the nav to be 'fixed' at the bottom. At the moment it will appear above on the favorite page.